### PR TITLE
Port the io command to plan9

### PR DESCRIPTION
**Pull request by rminnich**
We get both io and cmos querying commands.

### CODE REVIEW

File: pkg/memio/path_linux.go [hunk 1]
```diff
@@ -0,0 +1,7 @@
+// Copyright 2012-2019 the u-root Authors. All rights reserved
```
Comment (rjoleary) [inline pkg/memio/path_linux.go:1]: 2020

File: pkg/memio/ports_plan9.go [hunk 2]
```diff
@@ -0,0 +1,39 @@
+// Copyright 2012-2019 the u-root Authors. All rights reserved
```
Comment (rjoleary) [inline pkg/memio/ports_plan9.go:1]: 2020

File: pkg/memio/seek_linux.go [hunk 3]
```diff
@@ -0,0 +1,19 @@
+// Copyright 2012-2019 the u-root Authors. All rights reserved
```
Comment (rjoleary) [inline pkg/memio/seek_linux.go:1]: 2020

File: cmds/core/io/io_linux.go [hunk 4]
```diff
@@ -0,0 +1,25 @@
+// Copyright 2010-2019 the u-root Authors. All rights reserved
```
Comment (rjoleary) [inline cmds/core/io/io_linux.go:1]: 2020

File: cmds/core/io/io.go [hunk 5]
```diff
@@ -36,29 +36,24 @@ import (
...
 
+type cmdFunc func(addr int64, data memio.UintN) error
 type cmd struct {
-	f                 func(addr int64, data memio.UintN) error
+	f                 cmdFunc
 	addrBits, valBits int
 }
 
 var (
-	readCmds = map[string]cmd{
```
Comment (rjoleary) [inline cmds/core/io/io.go:45]: Plan9 has no way to read/write to physical memory?

Comment (rminnich) [reply cmds/core/io/io.go:45]: no, it's a security hole and was hence never allowed. The IOport thing is a bit of a hole too but we've let it go.

**Pull request merged**